src/Popover/Popover.js

```diff
@@ -78,13 +79,22 @@ class Popover extends Component {
             onClickFunctions = chain(this.triggerBody, control.props.onClick);
         }
 
-        const referenceComponent = React.cloneElement(control, {
-            onClick: onClickFunctions,
-            tabIndex: 0,
-            role: 'button',
-            'aria-haspopup': true,
-            onKeyPress: (event) => this.handleKeyPress(event, control, onClickFunctions)
-        });
+        let controlProps = {
+            onClick: onClickFunctions
+        };
+
+        if (!disableKeyPressHandler) {
+            controlProps = {
```

Comment:
Is it unsafe to add  `tabindex:0` and `aria-haspopup:true` when `disableKeyPressHandler` is `false/unset`?

Reply:
I think it may be, as these controls are compound components (such as ComboboxInput where the input is both a FormInput and a Button combined). None of the components I gave this prop to use anything besides a Button to trigger the Popover, so they're still accessible.